_includes/config/secure-ws-apache_step1.md

```diff
@@ -18,8 +21,10 @@ First, see if you have the Apache `htpasswd` utility is installed as follows:
 
 Enter the following commands as a user with `root` privileges:
 
-	mkdir -p /usr/local/apache/password
-	htpasswd -c /usr/local/apache/password/.<password file name> <username>
+```bash
+mkdir -p /usr/local/apache/password
+htpasswd -c /usr/local/apache/password/.<password file name> <username>
```

Comment:
Separate lines here.

Larger question. Maybe the rendering for `bash` can be adjusted so that every line within the code block gets a `$`?

Reply:
I was wondering the same thing just this morning.

Reply:
1. Each command executes different Unix tools, so each one expects some return and should be described separately.
2. When I _Copy_ the script, I want control it, and run step-by-step, not all at once. I also may want to edit each command with my data.

When you're running more than one command at a time, you should run it as a Shell script actually.